src/routes/(blog-article)/deploying-torrust-to-production/+page.md

```diff
@@ -0,0 +1,1073 @@
+- Scrape request: <https://tracker.torrust-demo.com/scrape?info_hash=%44%3C%76%02%B4%FD%E8%3D%11%54%D6%D9%DA%48%80%84%18%B1%81%B6>
+
+The infohash used is `443C7602B4FDE83D1154D6D9DA48808418B181B6`.
+
+## Maintenance
+
+TODO
+
+### Backups
+
+TODO
+
+### Monitoring
+
+TODO
+
+## Cost
+
+TODO
+
+## Other Considerations
+
+TODO
+
+- Add your tracker to <https://newtrackon.com/>
+
+## Links
+
+- [How To Secure a Containerized Node.js Application with Nginx, Let's Encrypt, and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-secure-a-containerized-node-js-application-with-nginx-let-s-encrypt-and-docker-compose)
+- [Initial Server Setup with Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-22-04)
+- [How to Set Up SSH Keys on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys-on-ubuntu-22-04)
+- [How To Install and Use Docker on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-22-04)
+- [How To Install and Use Docker Compose on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-22-04)
+- [How To Set Up a Firewall with UFW on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-with-ufw-on-ubuntu-22-04)
+- [How To Install Git on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-install-git-on-ubuntu-22-04)
+- [How To Set Up a Firewall with UFW on Ubuntu 22.04](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-with-ufw-on-ubuntu-22-04)
+- [BEP 34 DNS Tracker Preferences](https://www.bittorrent.org/beps/bep_0034.html)
+- <https://letsencrypt.org/>
+- <https://certbot.eff.org/>
+- [UDP Tracker with Nginx proxy](https://github.com/torrust/torrust-tracker/issues/)11
+
+## Conclusion
+
+With containers having a production ready Index and Tracker with Torrust is very easy. Containerization allows you to install the application in many different hosting providers with no effort.
+
+It also allows you to keep your production environment up to date, because it's also possible to easily update the docker images.
+
+And finally, you can also scale up your production environment as long as the demand increases. For example, just increasing the droplet size.
+
+We hope you find this guide useful. Let us know what you think about it!
+
+If you see something wrong or you want to contribute by:
+
+- Adding new sections.
+- Fixing typos.
+- Making it clearer.
+- Adding links.
+- Or whatever you think it could be interesting to have,
+
+please open an [issue](https://github.com/torrust/torrust-website/issues) or a [PR](https://github.com/torrust/torrust-website/pulls).
+
+If you have any questions or issues please open an issue on the corresponding repository:
```

Comment:
I think here "in" is the correct preposition rather then "on"

To open an issue in a repository.....

Reply:
Fixed